Вселенная RimWorld/cryptosleep_revival_briefing_trans.txt

```diff
@@ -0,0 +1,147 @@
+  * Двигатель Джонсона-Танака (Johnson-Tanaka Drive): Двигательная система космического аппарата, работающая без реактивной тяги. Это подразумевает, что для ускорения не нужно выбрасывать газ из задней части аппатата, подобно ракете, что позволяет ускоряться в течении многих лет. Эта технология в комбинации с криптосном - то, что делает межзвёздные путешествия возможными для живых людей. Двигатель не нарушает законы сохранения: он работает от передачи импульса ближайщим звёздам через точно нацеленные «пучки» (precisely-aligned “beams”) импульсных волн (momentum waves), которые можно представить как (instantiated in) экзотические виртуальные частицы.
+  * Механиты: Микроскопические механоиды. Наиболее известны по их применению в медицине, но могут быть запрграммированы на выполнение и других задач. Безопасное использование механитов подразумевает строгое предотвращение их самовоспроизведения.
+  * Энергетическое оружие (Charged-shot weapons, прим.пер. правильнее "зарядовое оружие", так как использует не плазму/лазер, а физический объект, окутанный в ЭМ поле, к тому же для энергооружия есть свой термин - energy weapon): Это оружие стреляет снарядами (projectiles), окутанными в матрицу из содержащих магнитное поле заряженных частиц (magnetically-contained charged particles). При соприкосновении энергия в частицах высвобождается в очень мощный взрыв.
+  * Чипы счастья (Joywires, прим.пер. неологизм, образованный от hard-, software; перевести можно как Бог на душу положит: чипы удовольствия, услаждатели, чипы радости etc): Крошечные электронные устройства, имплантируемые в мозг. Они стимулируют его, используя электричество и малые дозы химических веществ, как правило для создания эффекта эйфории. Вызывают сильное привыкание.
+
+    Биология человечества
+
+В записях Ордо История перечислены тысячи задокументированых контактов с инопланетной жизнью. Однако, при тщательном расследовании каждого случая инквиры Ордо (прим.пер. лат inquier спрашивать, выяснять) обнаружили, что инопланетяне, по сути, просто еще одна ветвь человечества.
+Помимо технологического многообразия людей, существует и широкое биологическое многообразие. Некоторые популяции эволюционировали под селекционным давлением доиндустриальной жизни или в свехгорячих, либо сверххолодных мирах, в мирах с высокой, либо низкой гравитацией, или даже в мирах, омытых токсичными осадками гиперразрушительных войн. Хотя почти во всех подобных ксенолюдях (как их называют) распознаваемо происхождение от людей Земли (original Earth stock), их морфология весьма отличается. Одни - гиганты, другие - крошечные, либо приземистые. Одни тёмные, другие — белые, как снег. Одни мохнатые, как звери, другие - совершенно безволосые. Питание, склонности, химическая и радиационная терпимость (tolerances) существенно варируются.
```

Comment:
http://gramota.ru/slovari/dic/?lop=x&word=%D0%B2%D0%B0%D1%80%D1%8C%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D1%82%D1%8C%D1%81%D1%8F